Tests/scripts/utils/content_packs_util.py

```diff
@@ -41,14 +42,32 @@ def is_pack_deprecated(pack_path: str) -> bool:
     return pack_metadata.get('hidden', False)
 
 
-def should_test_content_pack(pack_name: str) -> Tuple[bool, str]:
+def get_pack_supported_marketplace_version(pack_name: str, id_set: dict) -> list:
+    """Checks the supported marketplace versions.
+
+    Args:
+        pack_name (str): The pack name
+        id_set (dict): Structure which holds all content entities to extract pack names from.
+
+    Returns:
+        list of supported marketplace versions
+    """
+    if id_set:
```

Comment:
Are we expecting not to have id set in some cases?

Reply:
no, but for an unexpected case.